src/databricks/labs/ucx/hive_metastore/tables.scala

```diff
@@ -52,8 +53,13 @@ def metadataForAllTables(databases: Seq[String], queue: ConcurrentLinkedQueue[Ta
   }).toList.toDF
 }
 
-dbutils.widgets.text("inventory_database", "ucx")
-val inventoryDatabase = dbutils.widgets.get("inventory_database")
+dbutils.widgets.text("config", "config.yml")
+val configFile = dbutils.widgets.get("config")
+val fs = FileSystem.get(new java.net.URI("file:/Workspace"), sc.hadoopConfiguration)
+val file = fs.open(new Path(configFile))
+val configContents = org.apache.commons.io.IOUtils.toString(file, java.nio.charset.StandardCharsets.UTF_8)
+val configObj = new Yaml().load(configContents).asInstanceOf[java.util.Map[String, Any]]
+val inventoryDatabase = obj.get("inventory_database")
```

Comment:
can you refactor this into a single function, that returns just `inventoryDatabase`, so that we don't pollute global namespace